compiler/src/language_server/protocol.rei

```diff
@@ -154,3 +175,5 @@ let notification: (~method: string, Yojson.Safe.t) => unit;
 
 let uri_to_filename: uri => string;
 let filename_to_uri: string => uri;
+
+let loc_to_range: Grain_parsing.Location.t => range;
```

Comment:
This isn't really the "protocol" but instead a grain-to-protocol translation. It's probably fine here for now but we'll need to refactor in the future.

Reply:
Agreed, I moved it (and other utilities) to a Utils module.  Is that more what you were looking for?